db/migrate/20160609080700_add_clp_datamodel.rb

```diff
@@ -0,0 +1,19 @@
+class AddClpDatamodel < ActiveRecord::Migration
+  def up
+    create_table :landingpages do |t|
+      t.column :community_id, :integer, null: false
+      t.column :version, :integer, null: false
+      t.column :released, :datetime
+      t.column :content, :text, limit: 16.megabytes - 1, null: false
```

Comment:
I'm not adding default timestamps because this data is never written via AR. It's a good point still that should I add a last_updated field and manually update that too. Hmm...